driver_go18_test.go

```diff
@@ -293,12 +360,33 @@ func TestContextCancelQuery(t *testing.T) {
 		dbt.mustExec("CREATE TABLE test (v INTEGER)")
 		ctx, cancel := context.WithCancel(context.Background())
 
-		// Delay execution for just a bit until db.ExecContext has begun.
+		var longProcess *dbProcess
+		longQuery := "INSERT INTO test VALUES (SLEEP(1))"
```

Comment:
this is query, not exec. Please do not use copy-paste.
You can move this check into separate function, I mean check, that tests query cancelation.
and there should be option, it is exec query or select query.

Reply:
fixed